src/components/DayPicker.jsx

```diff
@@ -898,3 +813,110 @@ export default class DayPicker extends React.Component {
 
 DayPicker.propTypes = propTypes;
 DayPicker.defaultProps = defaultProps;
+
+export { DayPicker as PureDayPicker };
```

Comment:
i think i called this out before; this is just to spy on instance methods in tests, yes?

Reply:
Yes